compiler-core/src/javascript/tests/bit_arrays.rs

```diff
@@ -122,11 +122,27 @@ fn go() {
 }
 
 #[test]
-fn sized_big_endian() {
+fn sized_with_compile_time_evaluation() {
```

Comment:
Undo the changes to existing files. Tests should be added to, not deleted or edited.